acceptance/allocator_terraform/load.tf

```diff
@@ -3,6 +3,11 @@
 variable "example_block_writer_instances" {
   default = 1
 }
+
+# Despite its name, example_block_writer actually has `photos` too. We're
+# sticking to this name for compatibility with Terrafarm. Since our load
```

Comment:
https://github.com/cockroachdb/cockroach/blob/master/acceptance/terrafarm/farmer.go#L56

On Wed, Jun 22, 2016 at 11:49 AM Tobias Schottdorf <notifications@github.com>
wrote:

> In acceptance/allocator_terraform/load.tf
> <https://github.com/cockroachdb/cockroach/pull/7393#discussion_r68079523>:
>
> > @@ -3,6 +3,11 @@
> >  variable "example_block_writer_instances" {
> >    default = 1
> >  }
> > +
> > +# Despite its name, example_block_writer actually has `photos` too. We're
> > +# sticking to this name for compatibility with Terrafarm. Since our load
>
> Interesting, didn't know terrafarm made assumptions on this - what could
> you point one out in the code?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/7393/files/3b1b4cb06df7ac5443592eddfc6aad6fe47966b5#r68079523>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABffpu_pERlAI7PKJZ8FTEXCE31EtQvNks5qOVmMgaJpZM4I77Zy>
> .
>